variables.tf

```diff
@@ -41,11 +55,17 @@ variable "scaleway_region" {
   default = "ams1"
 }
 
-/* digitalocean */
+
+/* DIGITALOCEAN  */
+# see API docs for possible inputs: https://developers.digitalocean.com/documentation/v2/
+
+# export TF_VAR_digitalocean_token=$(cat /my/secret/tokens/digitalocean_token.txt)
 variable "digitalocean_token" {
   default = ""
 }
 
+# These are the names you give your key after uploading it to the provider.
+# export TF_VAR_digitalocean_ssh_keys='["My Desktop", "My Laptop"]'
```

Comment:
These can be ids too.

Reply:
What do you mean by ID's? Can we agree on an example that is a little more explicit? Matching the label of the key in the console [here](https://goo.gl/4hdRCF) is the only thing I found that worked. No offense, but `'["12548","17593"]'` is extremely vague, I don't know what these numbers are referencing.